naming/src/main/java/com/alibaba/nacos/naming/core/Cluster.java

```diff
@@ -312,7 +312,7 @@ public void updateIps(List<Instance> ips, boolean ephemeral) {
         
         Map<String, Integer> intersectMap = new ConcurrentHashMap<>(newInstance.size() + oldInstance.size());
         Map<String, Instance> instanceMap = new ConcurrentHashMap<>(newInstance.size());
-        Map<String, Instance> instanceMap1 = new ConcurrentHashMap<>(newInstance.size());
+        Map<String, Instance> instancesMap = new ConcurrentHashMap<>(newInstance.size());
```

Comment:
I read method again and I think we should change `instanceMap` to `updatedInstancesMap`
change `instanceMap1 ` to `newInstancesMap`

Reply:
Ok, I change